recipes/r-bootstrap/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = '2017.2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bootstrap
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: bootstrap_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/bootstrap_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/bootstrap/bootstrap_{{ version }}.tar.gz
+  sha256: 44f118c90ee226730175c467a16ac8d5b3169d610424e613da4f73348fd79522
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+  run:
+    - r-base
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('bootstrap')"  # [not win]
+    - "\"%R%\" -e \"library('bootstrap')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=bootstrap
+  license: BSD_3_clause
+  summary: Software (bootstrap, cross-validation, jackknife) and data for the book "An Introduction
+    to the Bootstrap" by B. Efron and R. Tibshirani, 1993, Chapman and Hall. This package
+    is primarily provided for projects already based on it, and for support of the book.
+    New projects should preferentially use the recommended package "boot".
+  license_family: BSD
+
+extra:
+  recipe-maintainers:
+    - cbrueffer
```

Comment:
If you like please include https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml